instat/static/InstatObject/R/instat_object_R6.R

```diff
@@ -2493,4 +2493,43 @@ DataBook$set("public", "get_data_entry_data", function(data_name, station, date,
 DataBook$set("public", "save_data_entry_data", function(data_name, new_data, rows_changed) {
   self$get_data_objects(data_name)$save_data_entry_data(new_data = new_data, rows_changed = rows_changed)
 }
-)
\ No newline at end of file
+)
+
+DataBook$set("public", "import_from_cds", function(user, dataset, elements, start_date, end_date, lon, lat, path, import = TRUE, new_name) {
+  all_dates <- seq(start_date, end_date, by = 1)
+  all_periods <- unique(paste(lubridate::year(all_dates), sprintf("%02d", lubridate::month(all_dates)), sep = "-"))
+  all_days <- c("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "23", "24", "25", "26", "27", "28", "29", "30", "31")
+  all_hours <- c("00:00", "01:00", "02:00", "03:00", "04:00", "05:00", "06:00", "07:00", "08:00", "09:00", "10:00", "11:00", "12:00", "13:00", "14:00", "15:00", "16:00", "17:00", "18:00", "19:00", "20:00", "21:00", "22:00", "23:00")
+  area <- c(lat[2], lon[1], lat[1], lon[2])
+  pb <- winProgressBar(title = "Requesting data from CDS", min = 0, max = length(all_periods))
+  nc_files <- vector(mode = "character", length = length(all_periods))
+  for (i in seq_along(all_periods)) {
+    y <- substr(all_periods[i], 1, 4)
+    m <- substr(all_periods[i], 6, 7)
+    curr_dates <- all_dates[month(all_dates) == as.numeric(m) & year(all_dates) == as.numeric(y)]
```

Comment:
This was the problem! Sorry forgot this. That's why it worked on my RStudio because I had `lubridate` loaded already.
```suggestion
    curr_dates <- all_dates[lubridate::month(all_dates) == as.numeric(m) & lubridate::year(all_dates) == as.numeric(y)]
```